class-wc-calypso-bridge-frontend.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Frontend functions for Calypso Bridge
+ * 
+ * Mostly related to the Customizer, Storefront, and its extensions
+ *
+ * @package WC_Calypso_Bridge_Frontend/Classes
+ * @since   1.0.0
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Frontend
+ */
+class WC_Calypso_Bridge_Frontend {
+	/**
+	 * Paths to assets act oddly in production
+	 */
+	const MU_PLUGIN_ASSET_PATH = '/wp-content/mu-plugins/wpcomsh/vendor/automattic/wc-calypso-bridge/';
+
+	/**
+	 * Plugin asset path
+	 *
+	 * @var string
+	 */
+	public static $plugin_asset_path = null;
+
+	/**
+	 * Class Instance.
+	 *
+	 * @var WC_Calypso_Bridge_Frontend instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'remove_storefront_default_footer_credit' ), 10 );	
+    }
+    
+    /**
+	 * Unhooks Storefront footer credit
+	 *
+	 */
+	public function remove_storefront_default_footer_credit() {
+		remove_action( 'storefront_footer', 'storefront_credit', 20 );
+		add_action( 'storefront_footer', array( $this, 'wpcom_ecommerce_plan_storefront_credit' ), 20 );
+	}
+
+	/**
+	 * Display the new WordPress.com like theme credit
+	 *
+	 * @since  1.0.0
+	 * @return void
+	 */
+	public function wpcom_ecommerce_plan_storefront_credit() {
+		?>
+		<div class="site-info">
+			<?php echo esc_html( apply_filters( 'storefront_copyright_text', $content = '&copy; ' . get_bloginfo( 'name' ) . ' ' . date( 'Y' ) ) ); ?>
```

Comment:
$content isn't used again, and this line is a bit hard to follow. does

`echo esc_html( apply_filters( 'storefront_copyright_text', '&copy; ' . get_bloginfo( 'name' ) . ' ' . date( 'Y' ) ) );`

work instead?